Assets/HoloToolkit-Examples/UX/Scenes/InteractableObjectExample.meta

```diff
@@ -0,0 +1,10 @@
+fileFormatVersion: 2
+guid: 801cf099a721ed941a68cbab6b058b2b
```

Comment:
Where is this meta file coming from?

Where is the accompanied data?

Reply:
Unity generated a scene folder while testing changes. Removed.